routes/wishCards.js

```diff
@@ -10,201 +10,259 @@ const router = express.Router();
-            wishItemName: req.body.wishItemName,
-            wishItemPrice: Number(req.body.wishItemPrice),
-            wishItemURL: req.body.wishItemURL,
-            childStory: req.body.childStory,
-            wishCardImage: req.file.location,
-            createdBy: res.locals.user._id
-        });
-        newCard.save((err) => {
-            if (err) console.log(err);
-        });
-        console.log(newCard);
-        res.redirect('/wishcards');
-    }
+  if (req.file === undefined) {
+    res.send(
+      'Error: File must be in jpeg, jpg, gif, or png format. The file \
+        must also be less than 5 megabytes.'
+    );
+  } else {
+    var newCard = new WishCard({
+      childFirstName: req.body.childFirstName,
+      childLastName: req.body.childLastName,
+      childBirthday: new Date(req.body.childBirthday),
+      childInterest: req.body.childInterest,
+      wishItemName: req.body.wishItemName,
+      wishItemPrice: Number(req.body.wishItemPrice),
+      wishItemURL: req.body.wishItemURL,
+      childStory: req.body.childStory,
+      wishCardImage: req.file.location,
+      createdBy: res.locals.user._id,
+    });
+    newCard.save((err) => {
+      if (err) console.log(err);
+    });
+    console.log(newCard);
+    res.redirect('/wishcards');
+  }
 });
 
 // @desc    Render wishCards.html which will show all wishcards
 // @route   GET '/wishcards'
 // @access  Public, all users can see
 // @tested 	Yes
 router.get('/', async (req, res) => {
-    try {
-        var results = await WishCard.find({});
-        console.log(results);
-        res.status(200).render('wishCards', {
-            user: res.locals.user,
-            wishcards: results
-        });
-    } catch (error) {
-        res.status(400).send(JSON.stringify({
-            success: false,
-            error: error
-        }));
-    }
+  try {
+    var results = await WishCard.find({});
+    res.status(200).render('wishCards', {
+      user: res.locals.user,
+      wishcards: results,
+    });
+  } catch (error) {
+    res.status(400).send(
+      JSON.stringify({
+        success: false,
+        error: error,
+      })
+    );
+  }
 });
 
 // @desc    search the wish cards by substring of wishItemName
 // @route   POST '/wishcards/search'
 // @access  Public, all users can see
 // @tested 	Yes
 router.post('/search', async (req, res) => {
-    try {
-        const results = await WishCard.find({
-            "wishItemName": {
-                "$regex": req.body.wishitem,
-                "$options": "i"
-            }
-        });
-        res.status(200).render('wishCards', {
-            user: res.locals.user,
-            wishcards: results
-        });
-    } catch (error) {
-        res.status(400).send(JSON.stringify({
-            success: false,
-            error: error
-        }));
-    }
+  try {
+    const results = await WishCard.find({
+      wishItemName: {
+        $regex: req.body.wishitem,
+        $options: 'i',
```

Comment:
double ticks got removed here is that an issue?

Reply:
Nah, double ticks aren't necessary there :+1: